service/src/test/java/uk/gov/hmcts/reform/fpl/service/ActionCmoServiceTest.java

```diff
@@ -11,43 +12,90 @@
 import uk.gov.hmcts.reform.document.domain.Document;
 import uk.gov.hmcts.reform.fpl.config.DocmosisConfiguration;
+import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration;
 import uk.gov.hmcts.reform.fpl.model.CaseManagementOrder;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.OrderAction;
 import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
-import uk.gov.hmcts.reform.fpl.service.config.LookupTestConfig;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
 
 import java.io.IOException;
+import java.time.LocalDateTime;
 import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createHearingBookingDynmaicList;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createHearingBookings;
 import static uk.gov.hmcts.reform.fpl.utils.DocumentManagementStoreLoader.document;
 
 @ExtendWith(SpringExtension.class)
 @ContextConfiguration(classes = {
-    JacksonAutoConfiguration.class, JsonOrdersLookupService.class, LookupTestConfig.class,
-    JsonOrdersLookupService.class, DateFormatterService.class, DirectionHelperService.class,
-    DocmosisConfiguration.class, RestTemplate.class, CaseDataExtractionService.class,
-    DocmosisDocumentGeneratorService.class, CommonCaseDataExtractionService.class, HearingBookingService.class,
-    HearingVenueLookUpService.class, DraftCMOService.class
+    JacksonAutoConfiguration.class, JsonOrdersLookupService.class, HearingVenueLookUpService.class,
+    DocmosisDocumentGeneratorService.class, RestTemplate.class, DocmosisConfiguration.class,
+    CommonCaseDataExtractionService.class, DateFormatterService.class, HearingBookingService.class,
+    DirectionHelperService.class
 })
 class ActionCmoServiceTest {
-    private final DraftCMOService draftCMOService;
+    private static final String LOCAL_AUTHORITY_CODE = "example";
+    private static final String COURT_EMAIL_ADDRESS = "FamilyPublicLaw+test@gmail.com";
+    private static final String COURT_NAME = "Test court";
+    private static final LocalDateTime MOCK_DATE = LocalDateTime.of(2018, 2, 12, 9, 30);
+    private final ObjectMapper mapper;
+    private final OrdersLookupService ordersLookupService;
+    private final HearingVenueLookUpService hearingVenueLookUpService;
+    private final CommonCaseDataExtractionService commonCaseDataExtractionService;
+    private final DocmosisDocumentGeneratorService docmosisDocumentGeneratorService;
+    private final DateFormatterService dateFormatterService;
+    private final HearingBookingService hearingBookingService;
+    private final DirectionHelperService directionHelperService;
+    private final HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration = new HmctsCourtLookupConfiguration(
+        String.format("%s=>%s:%s", LOCAL_AUTHORITY_CODE, COURT_NAME, COURT_EMAIL_ADDRESS));
```

Comment:
This was was moved into `LookupTestConfig`  so it can always be added to `ContextConfiguration` . Looks like you reverted? 
[LookupConfig Changes](https://github.com/hmcts/fpl-ccd-configuration/pull/655/files#diff-9c1c7175511872f2ebddee1f1c8ba079)